### DISCUSSION TITLE
Projects with datachannel

### DISCUSSION THREAD

**Discussion by foxnight2** [paullouisageneau/libdatachannel#1050] [General]
Do you have the projects to using datachannel ?

**Comment by paullouisageneau** [comment]
If you are looking for examples of existing projects using libdatachannel, here is a non-exhaustive list of open-source projects: [Geckos.io](https://github.com/geckosio/geckos.io), [Godot Engine](https://github.com/godotengine/webrtc-native), [HTTP Toolkit](https://github.com/httptoolkit/httptoolkit-server), [libp2p](https://github.com/libp2p/js-libp2p), [libtorrent](https://github.com/arvidn/libtorrent), [OBS studio](https://github.com/obsproject/obs-studio), [Streamr](https://github.com/streamr-dev/network), [WebRTC SFU Sora](https://github.com/shiguredo/sora-c-sdk), [x-minecraft-launcher](https://github.com/Voxelum/x-minecraft-launcher)
